server/app/views/fileupload/FileUploadViewStrategy.java

```diff
@@ -45,6 +46,10 @@ public FormTag renderFileUploadFormElement(StorageUploadRequest request) {
         .withClasses(getUploadFormClass());
   }
 
```

Comment:
**suggestion:** Could you add a javadoc for this?

Reply:
done